openshift_scalability/content/centos-stress/Dockerfile

```diff
@@ -1,10 +1,16 @@
 # vim:set ft=dockerfile:
-FROM centos:latest
+FROM centos:7
 
-# Install required packages
-RUN yum install -y bc gcc git gnuplot go java-1.8.0-openjdk make \
-                   openssh-clients patch rsync tar unzip && \
-    yum localinstall -y https://dl.fedoraproject.org/pub/epel/6/x86_64/stress-1.0.4-4.el6.x86_64.rpm && \
+### Setup user for build execution and application runtime
+ENV APP_ROOT=/opt/stress
+ENV PATH=${APP_ROOT}/bin:${PATH} HOME=${APP_ROOT}
```

Comment:
Can we do all three environment variables on one line?

Reply:
Will change.